src/services/checkout.ts

```diff
@@ -287,26 +288,26 @@ export class CheckoutService {
     try {
       const rate = await getUSDCRate();
       const amount = Number((checkout.fundsAmountMoney.toUnit() / rate).toFixed(6));
-
-      const assetTransfer = await AssetTransfer.create({
+      const postFee = Number(checkout.postAmountFee.toUnit()/ rate);
+      assetTransfer = await AssetTransfer.create({
```

Comment:
Previously `assetTransfer` was defined twice in two scopes, within the try block and outside. The outer reference was used in the catch block, but that shouldn't have worked since there wasn't an assignment within. 

AssetTransfer status was changed in the catch block

This should fix that. If you want I can remove this as well. lmk